lib/multicell_location/Kconfig

```diff
@@ -20,7 +20,7 @@ if MULTICELL_LOCATION
 
 config MULTICELL_LOCATION_SERVICE_NRF_CLOUD
 	bool "nRF Cloud location service"
-	depends on MODEM_JWT
```

Comment:
When looking this now, I'd would be nice if a condition in this level of stack could be like following:
depends on NRF_CLOUD_REST || NRF_CLOUD_MQTT

-> meaning that MODEM_JWT dependency should be handled with in NRF_CLOUD_REST. But in this case, call to  nrf_cloud_jwt_generate() should be hidden within nrf_cloud_rest_cell_pos_get().  I.e. would require changes to nrf_cloud api for jwt.